js/packages/web/src/views/artworks/index.tsx

```diff
@@ -16,7 +16,7 @@ const { Content } = Layout;
 
 export const ArtworksView = () => {
   const { connected } = useWallet();
-  const { isLoading, pullAllMetadata, storeIndexer, pullItemsPage } = useMeta();
+  const { isLoading, pullAllMetadata, storeIndexer, pullItemsPage, isLoadingMetadata } = useMeta();
```

Comment:
```suggestion
  const { isFetching, pullAllMetadata, storeIndexer, pullItemsPage, isLoadingMetadata } = useMeta();
```

Reply:
It's from this PR: https://github.com/metaplex-foundation/metaplex/pull/1232/files